translation.json

```diff
@@ -0,0 +1,40 @@
+{
+  "loginDataSource": {
+    "instruction": "Enter your access details below",
+    "emailPlaceholder": "Enter your email",
+    "passwordPlaceholder": "Enter your password",
+    "loginError": "The details you entered don’t match our records. Please try again.",
+    "login": "Log in",
+    "loading": "Loading...",
+    "forgotPassword": "Forgot your password?",
+    "verifyInstruction": "To reset your password you must verify your email address.",
+    "emailNotFound": "Your email address cannot be found. Please try again.",
+    "backToLogin": "Back to Login",
+    "verifyEmail": "Verify email",
+    "hasVerificationCode": "I have a verification code",
+    "verifyCodeInstruction": "A verification code was sent to <span class=\"verify-user-email\"></span>.",
+    "verificationCodePlaceholder": "Enter verification code",
+    "verificationCodeError": "You entered the wrong code or the code expired. Please try again or request a new code.",
+    "resend": "Resend verification code",
+    "verify": "Verify",
+    "resendSuccess": "A new code was sent to your email.",
```

Comment:
resendCodeSuccess?

Reply:
I updated the keys to use more levels of nesting. The `resendSuccess` key should have more context as well now.